netbsd_sys/cdefs.h

```diff
@@ -6,7 +6,6 @@
 #define __BEGIN_DECLS
 #define __END_DECLS
 #endif
-#define __unused
```

Comment:
does it work if you instead add here
```c
#ifndef __unused
#define __unused
#endif
```

Reply:
I'll give it a try!

Reply:
It does not - I get the error wen compiling tty.c - since netbsd_sys/cdefs.h is the first include, `__unused` gets defined early on.

Later, tty.c includes fcntl.h, which includes bits/stat.h, which defines `struct stat` as having a field named `__unused`, which gets wiped out and causes the error.

Reply:
alright, thanks for testing!